### ISSUE TITLE
skype for web doesn't work

### ISSUE THREAD

**Issue by marc1uk** [QupZilla/qupzilla#2368]
Giving qupzilla a try. I'd really like to replace firefox, because it starts to lag after a while and is a resource hog, but I've yet to find a browser like it that *just works*. 
Qupzilla seems to be another browser with great acclaims, "super lightweight", but ultimately impractical because it just _can't do_ all the internet..
On my distribution (mint debian) the latest package in the repository is something like 1.2 - very out of date, so comments here are for the AppImage version. I tried to build from source, but the required Qt version of 5.8 also isn't in the repository, and i'm too busy to spend time on this now. 
As reported in other issues, some youtube videos don't work - it seems like an absence of proprietary codecs, which is one thing, but again correcting that seems to require not only building from source, but also building the Qt libraries from source - which sounds non-trivial. This is a lot of work to expect users to go to to make youtube work. I mean, i guess not that many people use youtube anyway, right?

Anyway, reporting as I couldn't find any other comments on it in the issues: Skype for web also doesn't work.  After signing in there's just an endless 'Loading...' logo. Can't seem to find any way to give other debug info, but maybe someone could look into it. For now though, it's back to firefox for me - i use skype for work, so unlike the cat videos, i can't just take that hit. 

Apologies if this post comes across as a complaint - i appreciate people working to produce some alternatives. I just hope one day I'll be able to run one of them and not have to sacrifice what sites I can use as a result.

**Comment by Lombress** [comment]
Skype for Web works for me on Opensuse Leap 42.2 and I've been able to watch any video on YouTube so far.

The problem with Qupzilla is the distros offer old versions of QtWebengine and of Qupzilla. I have Quzpilla 2.0.1 and Qtwebengine 5.6.1, and I can't print with Qupzilla, don't have spellcheck and can't watch videos with DRM (such as Netflix). But you can do all of this with the last version of Qtwebengine, that is available on rolling distros such as Arch Linux and Opensuse Thumbleweed. I guess next year everything will work. In the meanwhile I still use Qupzilla when I don't have to print anything, because Firefox and Chrome heat my CPU

**Comment by marc1uk** [comment]
I don't seem to have any QtWebengine installed (I'm using the AppImage version as i mentioned in my post) but the about page when i open qupzilla says I have QtWebEngine v5.8.0, and Qupzilla 2.1.2.  So it would seem my versions are newer than yours, and yet they don't work. (I might add slack also doesn't work. Same as skype, just sits me at the logging in screen endlessly.) 
It's the first time i've even heard of AppImage, but my impression was that it should contain everything the app needs. That seems like a great advantage for people like me on distros that prioritize stability over being bang up-to-date. Shame it doesn't seem to work.

**Comment by cranes-bill** [comment]
Appimage works to me:
![capture du 2017-07-28 20 53 15](https://user-images.githubusercontent.com/5161278/28729983-1acd8ea0-73d7-11e7-9559-d2cda01f8bd7.png)
May you try with clean profile please!

**Comment by marc1uk** [comment]
Hey cranes-bill, nice suggestion! With a clean profile I could access both, so I took another look through possible settings I had configured, and found that disabling local storage of html5 content was preventing skype and slack from logging in. With that enabled both work like a charm, so looks like this can be my new regular browser! 👍 

Youtube compatibility still says it's missing H.264 and MSE&H.264 (not sure if that's one or two things missing), but I can live with this.
